config/socket_windows.go

```diff
@@ -0,0 +1,5 @@
+package config
+
+func setTransportAddress(cfg *Config) {
+	cfg.TransportAddress = `\\.\pipe\manifoldco\torus`
```

Comment:
Fixed the backslash. Also confirmed it's [being created](http://screenshots.siphoc.com/2017-03-11-14-19-47.png) now.